packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -164,14 +164,6 @@ export class AppSyncSwiftVisitor extends AppSyncModelVisitor {
     );
     result.push(classDeclaration.string);
 
-    const extensionDeclaration = new SwiftDeclarationBlock().withName('Amplify').asKind('extension');
-    const extensionBody = dedent`
-    static func addDataStore() throws {
-      try Amplify.add(plugin: AWSDataStoreCategoryPlugin(modelRegistration: AmplifyModels()))
-    }`;
-    extensionDeclaration.withBlock(extensionBody);
-    result.push(extensionDeclaration.string);
-
```

Comment:
Are these changes meant to be in this PR?

Reply:
These are 2 separate changes in one PR. But will be merged as separate commits